JobRecommendationService.md

```diff
@@ -0,0 +1,131 @@
+# Job Recommendation service
+# Contents
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+  - [JobData](#jobdata)
+    - [Job](#job)
+  - [UserData](#userdata)
+  - [UserPreferences](#userpreferences)
+    - [PreferredLocation](#preferredlocation)
+- [Response Structure](#response-structure)
+- [Versioning](#versioning)
+
+----------------------------
+# Summary
+
+Given a job ID and, optionally, an email, and a set of user preferences, the Job Recommendation Service provides recommendations of relevant and similar jobs.
+
+The endpoint for Job Recommendation Service is at https://api.careerbuilder.com/core/recommendations/job.
+
+----------------------------
+# Request structure
+The service supports GET and POST requests to:
+https://api.careerbuilder.com/core/recommendations/job.
+
+OAUTH credentials are **required** and the reference is provided [here](https://apimanagement.cbplatform.link/#/oauth/faq).
```

Comment:
Sorry, I should have been more specific maybe, but I would reinforce that OAuth is standardized across CB and not specific to us.

>>> OAuth credentials are **required**. Documentation for CareerBuilder authentication is provided by [here](https://apimanagement.cbplatform.link/#/oauth/faq) by the Platform Software team.